exercises/concept/lasagna/lasagna.rb

```diff
@@ -1,13 +1,13 @@
 class Lasagna
   def remaining_minutes_in_oven(actual_minutes_in_oven)
-    raise 'Please implement the Lasagna#remaining_minutes_in_oven method'
+    raise 'Please implement the Lasagna#remaining_minutes_in_oven method, remove this line of code when implemented'
```

Comment:
What about the following:

```suggestion
    raise 'Please implement the Lasagna#remaining_minutes_in_oven method, remove this line of code to start implementing'
```

This makes it clear that you need to remove the statement and suggests to do it when you start implementing the method, which makes the most sense I feel.

That being said, there is indeed the fact that the student won't know what `raise` is when they start this exercise. The only real alternative would be to use a comment instead, which _is_ explained to the student. That might actually be a better solution.

Reply:
Doesn't the tests fail if this is not implemented?  I really think that the tests could communicate what needs to be done instead, rather than having this information in the supplied file.  We might give the scaffolding, the method names, the fact that they are not designed to pass should give us the communication the student needs, rather than raising explicitly from here.

Reply:
Which aligns with the second part of my comment: using comments.

Reply:
I was thinking to not even necessarily do the comments.  The tests would be "in your face" from the test file.

But it is another chance to remind the student to remove stale comments, and that is never a bad thing.